README.md

```diff
@@ -1,70 +1,122 @@
+   - To do this, input cd and add your folder location. You can add the folder location by dragging the folder to Git bash.
+    $ cd '/c/Users/Documents/My Git Project'
 
-### Code Splitting
+    - Go to the page of the repository that you want to clone.
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/code-splitting](https://facebook.github.io/create-react-app/docs/code-splitting)
+    - Click on “Clone or download” and copy the URL.
 
-### Analyzing the Bundle Size
+    - Use the git clone command along with the copied URL from earlier;
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size](https://facebook.github.io/create-react-app/docs/analyzing-the-bundle-size)
+    - <code>$ git clone https://github.com/TracyMuso/Ronin-Finance</code>
 
-### Making a Progressive Web App
+   - Press Enter;
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app](https://facebook.github.io/create-react-app/docs/making-a-progressive-web-app)
+# Tests and Configuration
 
-### Advanced Configuration
+To track linter errors locally follow these steps:
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
+Download all the dependencies run:
 
-### Deployment
+<code>
+npm install
+</code>
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
+**Track HTML linter errors run:**
 
-### `npm run build` fails to minify
+<code>
+npx hint .
+</code>
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
+**Track CSS linter errors run:**
+<code>
+npx stylelint "**/*.{css,scss}"
+</code>
+
+**Track JavaScript linter errors run:**
+
+<code>
+npx eslint .
+</code>
+
+# Usage
+- This can be used to track prices and performance of various cryptocurrencies
+
+# Install
+- Set up Github Actions
+- Create a react app
+- Set up linter > Lighthouse , Webhint ,Stylelint , Eslint.
+- Webpack bundle for javascript.
+- Jest for  testing
+
+## Author
+
+👤 **Tracy Musongole**
+
+- GitHub: [@githubhandle](https://github.com/TracyMuso)
+- Twitter: [@twitterhandle](https://twitter.com/tracy_muso)
+- LinkedIn: [@LinkedIn](https://linkedin.com/in/tracy-musongole)
+
+
+## 🤝 Contributing
+Contributions, issues, and feature requests are welcome!
+
+
+Feel free to check the [issues page](https://github.com/TracyMuso/Ronin-Finance/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
Your  README should contain an "Acknowledgements" section or similar that gives appropriate credit to [Nelson Sakwa](https://www.behance.net/sakwadesignstudio) the [author of the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/). 

👍🏾